src/modules/reporting/containers/reporting-dispute-markets.js

```diff
@@ -33,7 +33,7 @@ const mapStateToProps = (state, { history }) => {
     navigateToAccountDepositHandler: () => history.push(makePath(ACCOUNT_DEPOSIT)),
     outcomes: disputeOutcomes,
     account: loginAccount.address,
-    isForking: state.universe.forkEndTime && state.universe.forkEndTime !== '0',
+    isForking: (state.universe.forkEndTime && state.universe.forkEndTime !== '0') || false,
```

Comment:
@bthaile Why was this needed?

Reply:
IsForking is undefined if there isnt a forking market. Giving it a default value

Reply:
Got it.